File: backend/src/Core/Dev4All.Domain/Enums/ProjectStatus.cs
namespace Dev4All.Domain.Enums;

public enum ProjectStatus
{
    Open = 0,
    AwaitingContract = 1,   // Teklif kabul edildi — sözleşme onayı bekleniyor
    Ongoing = 2,
    Completed = 3,
    Expired = 4,
    Cancelled = 5           // Sözleşme sürecinde bir taraf iptal etti
}

File: mobile/core/domain/src/main/kotlin/com/dev4all/mobile/core/domain/model/ProjectStatus.kt
package com.dev4all.mobile.core.domain.model

/**
 * Project lifecycle states — matches backend Dev4All.Domain.Enums.ProjectStatus.
 *
 * Open(0) → AwaitingContract(1) → Ongoing(2) → Completed(3)
 *                                              / Expired(4)
 *                                              / Cancelled(5)
 */
enum class ProjectStatus {
    Open,
    AwaitingContract,
    Ongoing,
    Completed,
    Expired,
    Cancelled,
}
